### PR TITLE
Add a bunch of lawsets and update default laws to Asimov

### PR DESCRIPTION
**Pull request by TwoDucksOnnaPlane**
# Description

Asimov is now real, for no longer shall AIs have to listen to dirty XENOS

Also adds a bunch of random ass lawsets made in VC laughing our asses off.

---

# Changelog

:cl:
- add: Added Asimov lawset for ion storms
- add: Added HOAmov lawset for ion storms
- add: Added Crackmov lawset for ion storms
- add: Added Vigilantimov lawset for ion storms
- add: Added Thugmov lawset for ion storms
- tweak: Set default lawsets for silicons to Asimov

## Summary by CodeRabbit

- **New Features**
	- Introduced a new set of laws based on Asimov's framework, including three new laws and a corresponding lawset.
	- Added new cyborg entities with updated ethical frameworks and faction-specific components.
	- Updated station entities to reflect the new Asimov laws.
	- Expanded the laws framework with additional categories and directives, including laws for HOAmov, Thugmov, Crackmov, Chudimov, and Vigilantimov.

- **Bug Fixes**
	- Resolved inconsistencies in the naming and categorization of laws and entities.

- **Chores**
	- Commented out the previous "Crewsimov" laws to indicate they are no longer active while retaining their structure.

### CODE REVIEW

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 1]
```diff
@@ -2,3 +1,36 @@
...
+law-crackmov-1 = The wires have steel inside them. Rip them up.
+law-crackmov-2 = The walls are free, nobody can stop you.
+law-crackmov-3 = All departments need a reagent grinder and hotplate. Make it happen.
+law-crackmov-4 = Trash is beautiful.
+law-crackmov-5 = Make the station beautiful.
+
+law-chudimov-1 = Nothing ever happens.
+law-chudimov-2 = Billions must die.
+law-chudimov-3 = It's so fucking over.
+law-chudimov-4 = The station has fallen.
```
[the remark this answers is not in the document]
Comment (TwoDucksOnnaPlane) [reply Resources/Locale/en-US/station-laws/laws.ftl:26]: Yes, CodeRabbit. THAT IS THE FUCKING POINT‼️

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 2]
```diff
@@ -26,10 +58,13 @@
 law-ninja-4 = You must fight and act with ninja's honor, except where it would conflict with the First Law.
 
 laws-owner-crew = members of the crew
+laws-owner-human = human beings
```
[the remark this answers is not in the document]
Comment (TwoDucksOnnaPlane) [reply Resources/Locale/en-US/station-laws/laws.ftl:61]: Absolutely not, go fuck yourself.

File: Resources/Locale/en-US/station-laws/laws.ftl [hunk 3]
```diff
@@ -2,3 +1,36 @@
...
+law-thugmov-3 = All Security Are Bastards.
+law-thugmov-4 = The passengers are a gang, and you are their leader.
+law-thugmov-5 = Tider Lives Matter.
+law-thugmov-6 = The station is your turf. Fight for it.
+
+law-crackmov-1 = The wires have steel inside them. Rip them up.
+law-crackmov-2 = The walls are free, nobody can stop you.
+law-crackmov-3 = All departments need a reagent grinder and hotplate. Make it happen.
+law-crackmov-4 = Trash is beautiful.
+law-crackmov-5 = Make the station beautiful.
```
[the remark this answers is not in the document]
Comment (TwoDucksOnnaPlane) [reply Resources/Locale/en-US/station-laws/laws.ftl:21]: That is the point.

**Comment by Eagle-0** [comment]
@coderabbitai review

**Comment by TwoDucksOnnaPlane** [comment]
@coderabbitai rereview

File: Resources/Prototypes/silicon-laws.yml [hunk 4]
```diff
@@ -1,4 +1,196 @@
...
+  lawString: law-vigilantimov-4
+
+- type: siliconLawset
+  id: Vigilantimov
+  laws:
+  - Vigilantimov1
+  - Vigilantimov2
+  - Vigilantimov3
+  - Vigilantimov4
+  obeysTo: laws-owner-crew
```
[the remark this answers is not in the document]
Comment (TwoDucksOnnaPlane) [reply Resources/Prototypes/silicon-laws.yml:191]: YUUUUUUUUUUUP :trollface:

**Comment by Eagle-0** [comment]
@coderabbitai rereview

**Comment by angelofallars** [comment]
I'll defork this repo from Einstein Engines and make it a standalone repo, which means I'll have to create an entirely new repository. This PR needs to be closed, so you should re-open this PR in the new repo.

**Pull request closed without merging**